action.yml

```diff
@@ -5,14 +5,14 @@ inputs:
   unityVersion:
     required: false
     default: 'auto'
-    description: 'Version of unity to use for testing the project. Use "auto" to get from your ProjectSettings/ProjectVersion.txt'
+    description: 'Version of unity to use for testing the project. Use "auto" to get from your ProjectSettings/ProjectVersion.txt. ⚠️ If testing a Unity Package, this field is required and cannot be set to "auto".'
```

Comment:
Is there a way to detect that a Unity Package is tested instead of the regular case?

Perhaps we can just smoothen this over inside the auto strategy?

Perhaps something like this?
```ts
  if (input.packageMode) return
```

Reply:
I added a thrown error for this case on [line 109 of `input.ts`](https://github.com/game-ci/unity-test-runner/pull/164/files#diff-4d2973eda0a779e451cdd0304253311207a08a08e31ef58809cc738edd23d14dR108-R117):

```
    if (packageMode) {
      if (unityVersion === 'auto') {
        throw new Error(
          'Package Mode is enabled, but unityVersion is set to "auto". unityVersion must manually be set in Package Mode.',
        );
      }

      packageName = this.getPackageNameFromPackageJson(projectPath);
      this.verifyTestsFolderIsPresent(projectPath);
    }
```

Does this address the problem or no? Not totally sure if this is what you were going for. 

I'd like to add an "auto" strategy for package mode, but not until after the base feature is released.